frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1052,6 +1080,15 @@ impl<T: Config> Pallet<T> {
 		// upon arrival, thus we would then remove it here. Given overlay it is cheap anyhow
 		ensure!(winners.len() as u32 == desired_targets, FeasibilityError::WrongWinnerCount);
 
+		// ensure that the solution's score can pass absolute min-score.
+		let submitted_score = solution.score.clone();
+		ensure!(
+			Self::minimum_untrusted_score().map_or(true, |min_score|
```

Comment:
By having this check here, would it affect the weight of `submit_unsigned`? Or is this operation minuscule enough to not have any noticeable effect?

Reply:
insignificant to reweigh.